ansible/playbook/tasks/update.yml

```diff
@@ -150,9 +152,20 @@
         option: allow_loading_unsigned_plugins
         value: vertamedia-clickhouse-datasource
 
-    # TODO Create /srv/alertmanager/data and /srv/victoriametrics/data in pmm-managed
-    #      to support Docker-way (swap image) updates.
-    #      https://jira.percona.com/browse/PMM-7024
+    - name: Clickhouse logger         | Configure clickhouse logger
+      replace:
+        path: /etc/clickhouse-server/config.xml
+        regexp: '(<logger>[\s\S]*)</logger>'
+        replace: |-
+          <logger>
+                  <!-- Possible levels: https://github.com/pocoproject/poco/blob/develop/Foundation/include/Poco/Logger.h#L105 -->
+                  <level>trace</level>
+                  <console>1</console>
+              </logger>
+
+    # Create /srv/alertmanager/data and /srv/victoriametrics/data in pmm-managed
```

Comment:
I think it's a good idea to stay TODO here. I mean TODO word :)

It's helpful when you decided to reduce the technical dept and editors (for example vscode) can create a TODO list based on this word.

Reply:
I removed it because what used to be a TODO did get implemented, so it's no longer a TODO item.